hedera-node/hedera-app/src/test/java/com/hedera/node/app/fees/FeeCalculatorImplTest.java

```diff
@@ -47,9 +51,13 @@ public class FeeCalculatorImplTest {
     @Mock
     private TransactionBody txnBody;
 
+    @NonNull
+    private Function<SemanticVersion, SoftwareVersion> softwareVersionFactory;
+
     @BeforeEach
     void setUp() {
         feeData = new FeeData(FeeComponents.DEFAULT, FeeComponents.DEFAULT, FeeComponents.DEFAULT, SubType.DEFAULT);
+        softwareVersionFactory = ServicesSoftwareVersion::new;
```

Comment:
Nit:

```suggestion
    @NonNull
    private static final Function<SemanticVersion, SoftwareVersion> softwareVersionFactory = ServicesSoftwareVersion::new;

    @BeforeEach
    void setUp() {
        feeData = new FeeData(FeeComponents.DEFAULT, FeeComponents.DEFAULT, FeeComponents.DEFAULT, SubType.DEFAULT);
```

here and in other tests